sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -394,6 +394,18 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val ADAPTIVE_EXECUTION_LOG_LEVEL = buildConf("spark.sql.adaptive.logLevel")
+    .internal()
+    .doc("Configures the log level for adaptive execution logging. The value can be 'trace', " +
+      "'debug', 'info', 'warn', or 'error'. The default log level is 'trace'.")
```

Comment:
`The default log level is 'trace'` -> `The default log level is 'debug'` because the default is `debug` at line 407?